app/imports/components/apply/form-test.jsx

```diff
@@ -9,9 +9,25 @@ import { FormSchema } from '/imports/schemas/TestForm.js';
 
 // Import methods
 import { saveApplicationSection } from '/imports/api/methods.js';
+import { saveApplyPosition } from '/imports/api/methods.js';
 
 // Component
 export default class FormTest extends Component {
+  savePosition() {
+    saveApplyPosition.call({
+      position: this.props.type,
```

Comment:
yh `step` is better and makes more sense as a numerical value. step 3 step 5 etc